sqlhelper.py

```diff
@@ -0,0 +1,37 @@
+import sqlite3
+
+#database setup function
+def setup_database(dbfile):
+    conn = sqlite3.connect(dbfile)
+    c = conn.cursor()
+
+    c.execute('''CREATE TABLE IF NOT EXISTS garage_status
+                 (id INTEGER PRIMARY KEY AUTOINCREMENT,
+                  garage_names TEXT, 
+                  garage_fullness TEXT, 
+                  garage_addresses TEXT,
```

Comment:
```suggestion
```
we dont want to store the address every time we store the fullness, the address is static we can have that in a global variable somewhere in the server